app/src/entry-point.lisp

```diff
@@ -299,27 +305,27 @@ Copyright (c) 2016-2019 Rigetti Computing.~2%")
             *available-allocation-kinds*))))
 
 (defun process-options (&key version
```

Comment:
If you're gonna re-indent this block, move `version` to the next line.

Reply:
emacs made me do it